supervisor/stub/internal_flash.c

```diff
@@ -46,20 +46,22 @@ uint32_t supervisor_flash_get_block_count(void) {
     return 0;
 }
 
-void port_internal_flash_flush(void) {
-    return;
+mp_uint_t supervisor_flash_read_blocks(uint8_t *dest, uint32_t block, uint32_t num_blocks) {
+    return 0; // success
 }
 
-mp_uint_t supervisor_flash_read_blocks(uint8_t *dest, uint32_t block, uint32_t num_blocks) {
+mp_uint_t supervisor_flash_write_blocks(const uint8_t *src, uint32_t block_num, uint32_t num_blocks) {
     return 0; // success
 }
 
-bool supervisor_flash_write_block(const uint8_t *src, uint32_t block) {
-    return true;
+#if (0)
```

Comment:
If this section of code is not needed now, it's probably better to delete it.

Reply:
I am fine to remove it. My idea was to state it here for sake of completeness since the function prototypes are declared together in `supervisor/flash.h`. This piece of inactive code also could be kept and I may comment it accordingly. Does it make sense?

```c
void supervisor_flash_init_vfs(struct _fs_user_mount_t *vfs);
void supervisor_flash_flush(void);
void supervisor_flash_release_cache(void);
```